### PR TITLE
Fix linux warnings in generated C code

### PR DESCRIPTION
**Pull request by nsauzede**
PR to fix as much (linux) warnings in C code generated by V (using external compilation of tvintris as a vehicle)

### CODE REVIEW

File: vlib/builtin/int.v [hunk 1]
```diff
@@ -156,17 +156,14 @@ pub fn (n int) hex() string {
...
-	// QTODO
-	//count := C.sprintf(charptr(hex), '0x%'C.PRIx64, n)
-	count := C.sprintf(hex, '0x%x', n)
+	count := C.sprintf(charptr(hex), '0x%'C.PRIx64, n)
 	return tos(hex, count)
 }
 
 pub fn (n u64) hex() string {
-	len := if n >= 0 { n.str().len + 3 } else { 19 }
+	len := if n > 0 { n.str().len + 3 } else { 19 }
```
Comment (spytheman) [inline vlib/builtin/int.v]: Why this changes >= to just > ? 
Is that needed to remove a warning ?

Comment (nsauzede) [reply vlib/builtin/int.v]: well, yeah for sure the intent was to fix a warning
and yes, testing `if unsigned >= 0` generates a warning because indeed, it is useless because all unsigned are *ALWAYS* >= 0
Which means tests like thi resolve to `if true` condition

Now that I think about it, it's true that the change to `if unsigned > 0` looks suspicious, even if not totally wrong.
For sure, the warning is gone.
But the remaining question would be : what was the original author intent behind this test.
If we can answer that question, probably we could push another commit to remove the current suspicious change I added.
:+1:

Comment (spytheman) [reply vlib/builtin/int.v]: Thanks. I have overlooked the fact that the argument is u64 :-| .
